pouta_blueprints/models.py

```diff
@@ -237,17 +294,72 @@ def model(self, value):
         self._model = json.dumps(value)
 
 
+class BlueprintTemplate(db.Model):
+    __tablename__ = 'blueprint_templates'
+    id = db.Column(db.String(32), primary_key=True)
+    name = db.Column(db.String(MAX_NAME_LENGTH))
+    _config = db.Column('config', db.Text)
```

Comment:
as a whole I'm not a big fan of this style of property storing in databases but it seems to be the way things have been done in this project. Now is not the time to change it but suffice to say that I'd like to at some point.